### ISSUE TITLE
Unity based Modding Support with Steam Workshop support?

### ISSUE THREAD

**Issue by GeminiSquishGames** [naelstrof/KoboldKare#157]
I know @naelstrof has stated modding is pretty certain, but also, "...if I ever make it that far." #34 which is slightly disconcerting. I'm waiting to do any of my own development on the source but also like having the Steam version so I can get updates without having to wait for the rebuilding, and having that mod management would be awesome for both just having others mods and my own in an easy to manage and build for the game way.

I'm hoping that modding support will be rather seamless to work with the steam version as I like having the official steam version to play and my own Unity build to play with. Takes a while to build even just minor new changes for me and having a... *ahem* "small package" that I can work on rather than the whole source would be great, but I'm also really curious to see what others will do and have it available in Steams Mod management system. It's just a little more overwhelming working on the source, trying to find all the parts I need, and knowing or not if it really should be part of the main game and expecting already busy people to help push changes when there are already people working on the official development who are intimate with the source.

**Comment by naelstrof** [comment]
Yeah! Check out the modding-support branch, it's slow going but I've got most everything in the game converted into a mod. It's my number 1 priority right now to make it work well, and to be easy to work with.

**Comment by GreenboxGames** [comment]
Awesome! I know you sounded pretty positive and motivated in more recent posts about it. I just needed to lurk moar. Thanks for all your excellent work mans! I'll check it out if I can stop playing...
